components/App/Button.vue

```diff
@@ -35,14 +32,27 @@ const componentToRender = computed(() => props.to ? resolveComponent('AppLink')
   <component :is="componentToRender" :type="to ? undefined : 'button'" :class="selectedLook.all" :to="to"
     class="focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-pink-600 inline-block">
     <template v-if="selectedLook.needsPseudo">
-      <span :class="selectedLook.before" />
-      <span :class="'main' in selectedLook && selectedLook.main">
+     <article :class="[selectedLook.main,'newButton']">
         <slot />
-      </span>
-      <span :class="[selectedLook.after, secondaryAfterBg]" />
+      </article>
     </template>
     <template v-else>
       <slot />
     </template>
   </component>
-</template>
\ No newline at end of file
+</template>
+
+<style scoped>
```

Comment:
Hey! This is still not using TailwindCSS classes *inline*. I'd prefer not using a style block here. See e.g. other components